Makefile

```diff
@@ -19,7 +19,7 @@ all: generate license fix vet fmt test lint tidy
 	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.44.2
 
 "$(MYGOBIN)/deepcopy-gen":
-	go install k8s.io/code-generator/cmd/deepcopy-gen@v0.23.6
+	go install k8s.io/code-generator/cmd/deepcopy-gen@v0.24.1
```

Comment:
is this needed for this PR or just coincidental? It can probably be pulled out.

Reply:
deepcopy-gen needed to be bumped to generate the right methods for `Inventory` to satisfy `runtime.Object` interface. This will be handled in #606 and this PR will rebase to pull this change out.